docs/docsite/rst/community/steering/community_steering_committee.rst

```diff
@@ -117,7 +117,11 @@ When reviewing community collection `inclusion requests <https://github.com/ansi
 
 #. In the course of the review, the Committee member marks items as completed or leaves a comment saying whether the reviewer expects an issue to be addressed or whether it is optional (for example, it could be **MUST FIX:** <what> or **SHOULD FIX:** <what> under an item).
 
-#. For a collection to be included in the Ansible community package, the collection must be reviewed and approved by at least two Committee members.
+#. For a collection to be included in the Ansible community package, the collection:
+
+  * Must be reviewed and approved by at least two persons, where at least one person is a Steering Committee member.
+  * Non-Steering Committee reviews to be counted for inclusion must be checked and approved by another Steering Committee member.
```

Comment:
* A Non-Steering Committee review to be counted for inclusion must be checked and approved by another Steering Committee member.